workflows/Atlas-Scanpy-CellBrowser.ga

```diff
@@ -83,20 +83,7 @@
                     "output_name": "matrix_mtx"
                 }
             }, 
-            "inputs": [
-                {
-                    "description": "runtime parameter for tool Scanpy Read10x", 
-                    "name": "genes"
-                }, 
-                {
-                    "description": "runtime parameter for tool Scanpy Read10x", 
-                    "name": "matrix"
-                }, 
-                {
-                    "description": "runtime parameter for tool Scanpy Read10x", 
-                    "name": "barcodes"
-                }
-            ], 
+            "inputs": [], 
```

Comment:
Seeing this change for all steps. What do the contents of "input": [] actually do?

Reply:
Not sure. I have double check though and they seem to look the same and work very much the same.

Reply:
on the UI I mean.